musicbrainz/csv/convert_to_csv.py

```diff
@@ -28,13 +28,13 @@
 
 entity_type = sys.argv[2]
 inputpath = os.path.join(DIRNAME, sys.argv[1])
-outputpath = os.path.join(DIRNAME, "data", f"{entity_type}.csv")
+outputpath = os.path.join(DIRNAME, "../data/output", f"{entity_type}.csv")
```

Comment:
Why are you using slash separators within an `os.path.join` call? the whole point of `os.path.join` is that it adds OS-appropriate path separators for you!